src/ML-POD/pod.cpp

```diff
@@ -0,0 +1,3763 @@
+/***************************************************************************
+               CESMIX-MIT Project
+
+ Contributing authors: Ngoc-Cuong Nguyen (cuongng@mit.edu, exapde@gmail.com)
+ ***************************************************************************/
+
+// POD header file
+
+#include "pod.h"
+
+// LAMMPS header files
+
+#include "comm.h"
+#include "error.h"
+#include "memory.h"
+#include "tokenizer.h"
+
+#include <string>
+#include <vector>
+#include <iomanip>
+#include <limits>
+#include <glob.h>
+#include <random>
+#include <algorithm>
+
+#define _USE_MATH_DEFINES
```

Comment:
Please do not use this define and do not include math.h but cmath.
For math constants, you should be including the LAMMPS provided "math_const.h" header and the constants included therein. This way we have consistent math constants across platforms and C-library implementations.

Reply:
Removed _USE_MATH_DEFINES and math.h. We now use cmath and math_const.h.